trunk/SUAVE/Analyses/Stability/AVL.py

```diff
@@ -517,7 +519,7 @@ def evaluate_conditions(self,run_conditions):
 
         # write the input files
         with redirect.folder(run_folder,force=False):
-            write_geometry(self,spanwise_elements,chordwise_elements)
+            write_geometry(self)
```

Comment:
this is inconsistent with the one before

Reply:
I renamed it spanwise_vortices and chordwise_vortices so that it is consistent with other files in the code. I also appended it to self so you don't need to pass more than one argument into the write_geometry() function

Reply:
does this work if you use SU2 aero mixed with AVL stability?

Reply:
that is a good question. since AVL stability runs its own computations I don't see why it won't work. I've run Fidelity Zero Aero with AVL stability already